assignment1/src/main/java/ro/utcn/sd/he/assignment1/model/QuestionTag.java

```diff
@@ -0,0 +1,21 @@
+package ro.utcn.sd.he.assignment1.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class QuestionTag {
+    @Id
+    @Column(name = "questionID")
```

Comment:
Naming conventions: in Java you use camelCase (which is OK in your case), but on the DB you use low_snake names (i.e. `question_id` instead of `questionID`). That's because most DBs are case insensitive when it comes to identifiers.

Reply:
I'll try to keep this in mind :).